docs/code_navigation/references/indexers.mdx

```diff
@@ -8,16 +8,16 @@ This table is maintained as an authoritative resource for users, Sales, and Cust
 
 |       Language        |     Indexer     | Status | Hover docs | Go to definition | Find references | Cross-file | Cross-repository | Find implementations |  Build tooling   |
 | --------------------- | --------------- | ------ | ---------- | ---------------- | --------------- | ---------- | ---------------- | -------------------- | ---------------- |
-| Go                    | lsif-go         | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    | -                |
-| TypeScript/JavaScript | scip-typescript | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    | -                |
-| C/C++                 | scip-clang      | 🟡      | ✓          | ✓                | ✓               | ✓          | ✓                | ✗                    | [See notes](https://github.com/sourcegraph/scip-clang#supported-platforms)        |
-| Java                  | scip-java       | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✓                | ✓                    | [See notes](https://sourcegraph.github.io/scip-java/docs/getting-started.html#supported-build-tools)        |
-| Scala                 | scip-java       | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✓                | ✓                    | [See notes](https://sourcegraph.github.io/scip-java/docs/getting-started.html#supported-build-tools)        |
-| Kotlin                | scip-java       | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✗                | ✓                    | [See notes](https://sourcegraph.github.io/scip-java/docs/getting-started.html#supported-build-tools)        |
-| Rust                  | rust-analyzer   | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✗                | ✓                    | [See notes](https://rust-analyzer.github.io/)        |
-| Python                | scip-python     | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    | [See notes](https://github.com/sourcegraph/scip-python)        |
-| Ruby                  | scip-ruby       | 🟢      | ✓          | ✓                | ✓               | ✓          | ✗                | ✓                    | [See notes](https://github.com/sourcegraph/scip-ruby#scip-ruby)        |
-| C#                    | scip-dotnet     | 🟠      | ✓          | ✓                | ✓               | ✗          | ✓                | ✓                    | `.sln` and `.csproj` |
+| Go                    | [scip-go](https://github.com/sourcegraph/scip-go)         | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    |                |
```

Comment:
This table is much harder to read now in the un-rendered form. 😬 

![CleanShot 2024-03-20 at 10 01 35@2x](https://github.com/sourcegraph/docs/assets/93103176/842d5fb5-d224-4af3-b612-4f649319298b)

I think you could preserve the original formatting with extra `[]` brackets like so:

```
[scip-go][]

After the table:

[scip-go]: https://github.com/sourcegraph/scip-go
```

Reply:
@varungandhi-src Great suggestion, it looks much better now